docs/configuration.md

```diff
@@ -0,0 +1,212 @@
+# Configuration
+
+## terra.yaml
+
+The terra.yaml file is the only Terra-specific addition required. This file defines inputs that the web 
+UI uses to display storage options, connection settings, and configuration parameters.
+
+```yaml linenums="1" title="Example terra.yaml structure"
+--8<-- "plugins/deadline10/terra.yaml"
+```
+
+#### Plugin Metadata
+
+| Key         | Type             | Description                                                                             |
+|-------------|------------------|-----------------------------------------------------------------------------------------|
+| resource_id | String           | (Deprecated)                                                                            |
+| name        | String           | Release Name used to identify the install                                               |
+| icon        | String           | http(s) link to the icon image                                                          |
+| description | String           | Brief description of your plugin                                                        |
+| category    | String           | The world your plugin is a part of                                                      |
+| tags        | Array of Strings | Tags are used for searching and informing Terra what this Plugin is a part of           |
```

Comment:
It may be good to reference some of the App Store integration around "cluter-level" as a tag. 

Not sure if we want that here,or may another section just going over some of those "integration" type of things.

Reply:
Added